addons/account_edi_ubl/models/account_move.py

```diff
@@ -11,6 +11,7 @@ def _get_ubl_values(self):
         def format_monetary(amount):
             # Format the monetary values to avoid trailing decimals (e.g. 90.85000000000001).
             return float_repr(amount, self.currency_id.decimal_places)
+        self.ensure_one()
```

Comment:
IMO, this way is more clean:
```
def _get_ubl_values(self):
    self.ensure_one()
    
    def format_monetary(amount):
            # Format the monetary values to avoid trailing decimals (e.g. 90.85000000000001).
            return float_repr(amount, self.currency_id.decimal_places)
    
    return ...
```